gulp/tasks/release-bower.js

```diff
@@ -0,0 +1,77 @@
+var gulp = require('gulp');
+var runSequence = require('run-sequence');
+var bump = require('gulp-bump');
+var gutil = require('gulp-util');
+var git = require('gulp-git');
+var fs = require('fs');
+var argv = require('yargs').argv;
+var config = require('../config').browserify;
+var uglify = require('gulp-uglify');
+var rename = require('gulp-rename');
+var currentPackage = getBowerJSONversion();
+
+gulp.task('build-dist', ['browserify'], function() {
+  return gulp.src(config.bundleConfigs[0].productionFolder + '/' + config.bundleConfigs[0].outputName) 
+    .pipe(gulp.dest(config.bundleConfigs[0].distributionFolder))
+    .pipe(uglify())
+    .pipe(rename({ extname: '.min.js' }))
```

Comment:
This tasks differs slightly from production-browser task, production-browser task only creates one file (minificated) and this creates two. Also we are using here different output paths because we want to create the output files in a different folder (`./dist`). There should be a different way to approach the same behaviour with only one task but I haven't found it.